internal/kibana/prefixes.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+const (
+	// CoreAPI is the prefix for all Kibana Core API resources.
+	CoreAPI = "/api/kibana"
```

Comment:
WDYT about renaming it to `api.go` or `url_prefixes.go`? Prefixes sounds a bit vague.

Reply:
Done in 93223e5.